contracts/actions/AaveDeposit.sol

```diff
@@ -51,25 +49,45 @@ contract AaveDeposit is IAaveDeposit {
 
         shares = aToken.scaledBalanceOf(address(this)) - balanceBefore;
 
-        id = _updateAavePosition(token, shares, tokenId);
-        emit DepositedOnAave(address(this), token, id, shares);
+        _updateAavePosition(token, shares, tokenId);
+        emit DepositedOnAave(address(this), token, tokenId, shares);
     }
 
+    ///@dev now dont use anymore aaveid since we have tokenId that is unique - store shares for tokenId and tokenToAave for tokenId
```

Comment:
if we write _**can't**_ let's write _**don't**_

Reply:
also, is it correct to use natspec to clarify updates in the code? Shouldn't it be readable for someone who looks at the code for the first time?